### ISSUE TITLE
Add friendly message when current user does not have permission to view the configured Activiti app

### ISSUE THREAD

**Issue by wabson** [Alfresco/alfresco-ng2-components#734]
Both tasklist and processlist can be configured to only display items from a specific app in Activiti. The Activiti API returns a 403 when process instance filters are requested for an app that the current user does not have permission to see, but the component does not display an text indicating what the problem is, just an empty list.

**Comment by magemello** [comment]
the problem doesn't happen if you use the activiti-apps component to select the app.

**Comment by wabson** [comment]
No, it doesn't, but in the sales-translation example app the idea is to hard-code the appId rather than allowing the user to select any app they like. So in this case a more friendly error is needed.

**Comment by magemello** [comment]
@wabson is this issue still valid?

**Comment by wabson** [comment]
@magemello I think so, if anything more so now we more use of router. I'll check today to see if it is actually still an issue, though.

**Comment by wabson** [comment]
Reproduced this using the demo shell quite easily:

1. Click into an app that you have created and published and take that URL (will be something like http://localhost:3000/activiti/apps/2003/tasks)
2. Log out and back in as a different user who has not been granted permission to that app
3. Navigate to the previous app using the URL

It is possible to at least give the user a clue to what has happened by making use of the `onError` events emitted by the components, similar to what we already do for 'Files'. I was able to achieve this with the addition of a few lines to the `activiti-demo` component inside the demo shell as shown in the following screenshot

<img width="1416" alt="screen shot 2017-01-13 at 16 15 28" src="https://cloud.githubusercontent.com/assets/1727487/21936999/502fa552-d9ad-11e6-9cee-44f037522705.png">
